control-plane/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/hashicorp/consul-k8s/control-plane/cni v0.0.0-20220831174802-b8af65262de8
 	github.com/hashicorp/consul-server-connection-manager v0.1.0
-	github.com/hashicorp/consul/api v1.10.1-0.20230331190547-fc64a702f43c
```

Comment:
I had to run `go mod tidy` to get this to work

Reply:
Also I think your tests are failing, when I click on the tests that are still "pending" they show as failed